cdap-proto/src/main/java/co/cask/cdap/proto/RunRecord.java

```diff
@@ -35,15 +37,27 @@
   @SerializedName("status")
   private final ProgramRunStatus status;
 
-  public RunRecord(String pid, long startTs, Long stopTs, ProgramRunStatus status) {
+  @SerializedName("adapter")
+  private final String adapterName;
```

Comment:
May not be a good idea to include adapter in this class, since all program runs will start showing adapter in the run records whether they were started from adapter or not, right? Possible to have class `AdapterRunRecord` which would extend the `RunRecord` to include the adapter info in it?